<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>620</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>620</height>
   </size>
  </property>
  <property name="font">
   <font>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>850</width>
      <height>620</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>850</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 85, 255, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>841</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
font: 16pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>841</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Baskerville Old Face</family>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Save credentials for the first time to proceed. You can also update them.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>Update Credentials </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="back_settings">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#back_settings{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
}

QPushButton#back_settings:hover{
background-color: rgba(0,0,0,0.9)
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="twillio_phone">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Twillio sms client phone number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="twillio_token">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>TWILIO AUTH TOKEN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="twillio_sid">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>TWILIO ACCOUNT SID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_crd">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>300</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#save_crd{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
}

QPushButton#save_crd:hover{
background-color: rgba(0,0,0,0.9)
}</string>
     </property>
     <property name="text">
      <string>Save/Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_crd">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>410</y>
       <width>300</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#clear_crd{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
}

QPushButton#clear_crd:hover{
background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Delete Configuration</string>
     </property>
    </widget>
    <widget class="QLabel" name="set_status">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>621</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
